rom/dev/tests/test_image_validation.rs

```diff
@@ -1339,6 +1339,12 @@ fn cert_test_with_custom_dates() {
     hw.upload_firmware(&image_bundle.to_bytes().unwrap())
         .unwrap();
 
+    hw.step_until_output_contains("[exit] Launching FMC")
+        .unwrap();
+
+    let result = hw.mailbox_execute(0x1000_0001, &[]);
+    assert!(result.is_ok());
```

Comment:
nit: Prefer `hw.mailbox_execute(0x1000_0001, &[]).unwrap();`

Reply:
Or assert_eq!(hw.mailbox_execute(0x1000_0000), Ok(None)) if you want to be certain it didn't send you back any response data.

Reply:
The advantage of these approaches over is_ok() is that when something goes wrong it will log the error code.

Reply:
Updated.